CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,96 @@
+
+This Code of Conduct outlines our expectations for all those who participate in
+our projects and community, as well as the consequences for unacceptable
+behavior.
+
+We invite all those who participate in our projects to help us create safe and
+positive experiences for everyone.
+
+## Open [Source/Culture/Tech] Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open
+[source/culture/tech] citizenship by encouraging participants to recognize and
+strengthen the relationships between our actions and their effects on our
+community.
+
+Communities mirror the societies in which they exist and positive action is
+essential to counteract the many forms of inequality and abuses of power that
+exist in society.
+
+If you see someone who is making an extra effort to ensure our community is
+welcoming, friendly, and encourages all participants to contribute to the
+fullest extent, we want to know.
+
+## Expected Behavior
+
+* Participate in an authentic and active way. In doing so, you contribute to the
+  health and longevity of this community.
+* Exercise consideration and respect in your communication and actions.
+* Attempt collaboration before conflict.
+* Give constructive feedback.
+* Refrain from demeaning, discriminatory, or harassing behavior.
+* Be mindful of your fellow participants. Alert the project leaders if you
+  notice violations of this Code of Conduct, even if they seem inconsequential.
+
+## Unacceptable Behavior
+
+Unacceptable behaviors include: intimidating, harassing, trolling, abusive,
+discriminatory, derogatory or demeaning communication or actions by any
+participant in our community online, at all related projects and in one-on-one
+communications carried out in the context of community business.
+
+
+Harassment includes: harmful or prejudicial verbal or written comments related
+to gender, sexual orientation, ethnicity, religion, disability; inappropriate
+use of nudity and/or sexual images; inappropriate depictions of violence;
+deliberate intimidation, stalking; unwelcome sexual attention.
+
+## Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including corporate
+participants or sponsors and those with decision-making authority, will not be
+tolerated. Anyone asked to stop unacceptable behavior is expected to comply
+immediately.
+
+If a community member engages in unacceptable behavior, the project maintainers
+may take any action they deem appropriate, up to and including a
+temporary or permanent ban from participating in the project and community
+without warning.
+
+## If You Witness or Are Subject to Unacceptable Behavior
+
+If you are subject to or witness unacceptable behavior, or have any other
+concerns, please report it to conduct@flatcar-linux.org.
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, you should notify us at conduct@flatcar-linux.org with a concise
```

Comment:
make sure this email is active before publishing (it's currently not)

Reply:
Done (well, @iaguis did it).